docs/_docs/containers-asdkviewcontroller.md

```diff
@@ -0,0 +1,67 @@
+---
+title: ASDKViewController
+layout: docs
+permalink: /docs/containers-asdkviewcontroller.html
+prevPage: asrunloopqueue.html
+nextPage: containers-asnodecontroller.html
+---
+
+`ASDKViewController` is a subclass of `UIViewController` that adds several useful features for hosting `ASDisplayNode` hierarchies.
+
+An `ASDKViewController` can be used in place of any `UIViewController` - including within a `UINavigationController`, `UITabBarController` and `UISplitViewController` or as a modal view controller. 
+
+Benefits of using an `ASDKViewController`:
+<ol>
+<li><b>Save Memory</b>. An <code>ASDKViewController</code> that goes off screen will automatically reduce the size of the <a href="intelligent-preloading.html">fetch data</a> and <a href="intelligent-preloading.html">display ranges</a> of any of its children. This is key for memory management in large applications. </li>
+<li><b><a href="asvisibility.html"><code>ASVisibility</code></a> Feature</b>. When used in an <code>ASNavigationController</code> or <code>ASTabBarController</code>, these classes know the exact number of user taps it would take to make the view controller visible.</li>
```

Comment:
`ASDKNavigationController`, not `ASNavigationController`

Reply:
This was just for ASDKViewController but I can update it to change ASDKNavigationController as well. Thanks.